runmariadb.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/bash
+PATH=$PATH:/usr/sbin/
+DATADIR="/var/lib/mysql"
+
+crudini --set /etc/my.conf mysqld max_connects 64
+crudini --set /etc/my.conf mysqld max_heap_table_size 1M
+crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
+crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
+
+mysql_install_db --datadir="$DATADIR"
```

Comment:
So what if a container is restarted? Do we really want to do this again?  Same with setting the password below, do we really want to in that case?

Reply:
I will add a logic for the restart